client/scss/components/_comments-controls.scss

```diff
@@ -1,52 +1,89 @@
 .comments-controls {
-    padding-right: $mobile-nice-padding;
+    position: relative;
+    display: flex;
+    justify-content: flex-end;
+    padding-right: 2px;
+    height: 42px;
+    padding-bottom: 8px;
+    box-sizing: border-box;
 
     @include media-breakpoint-up(sm) {
-        padding-right: $desktop-nice-padding;
+        padding-right: 30px;
+        height: 43px;
     }
 }
 
 .comments-toggle {
-    float: right;
+    $root: &;
+    float: none;
     position: relative;
-    width: 42px;
-    height: 42px;
     cursor: pointer;
+    display: flex;
+    width: auto;
+    align-items: center;
 
     // Wagtail adds some top padding to labels on mobile
     padding: 0;
 
+    &--active,
+    &:hover {
+        #{$root}__label {
+            opacity: 1;
+        }
+
+        #{$root}__icon {
+            color: $color-white;
+        }
+    }
+
     &__icon {
         position: absolute;
         left: -12px;
         bottom: 3px;
         width: 52px;
         height: 52px;
-        color: $color-white;
-        opacity: 0.6;
-        transition: opacity 100ms cubic-bezier(0.4, 0, 0.2, 1);
+        color: #004345;
```

Comment:
There's quite a few hardcoded colours in here - are none of them sufficiently close to the colour variables we've got?